client/components/ContactAgent.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import styled from 'styled-components';
+const ContactAgent = (props) => (
+  <React.Fragment>
+{/*  */}
+  <FormContainer>
+    <Form>
+      <FlexContainerRow>
+        <SimpleInput placeholder={'Name'} type={'text'} value={''}></SimpleInput>
+        <SimpleInput placeholder={'Phone'} pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}" type={'tel'} value={''} ></SimpleInput>
```

Comment:
Is pattern here a random phone number generator?

Reply:
It's supposed to control the pattern for the phone number, but it needs a bit more work.